scripts/start.sh

```diff
@@ -88,6 +88,7 @@ else
   echo ""
 fi
 
+dev_mode=$(python -c "import feconf; print feconf.DEV_MODE")
```

Comment:
Actually, this script is never run in non-dev mode, so I don't think you need this.

At some point we should write something that does e2e tests on minified files, but let's leave that to a separate commit.